### ISSUE TITLE
Develop baseline network to compare results

### ISSUE THREAD

**Issue by fedetask** [CampusAI/Hamiltonian-Generative-Networks#8]
In order to understand whether the Hamiltonianapproach is effectively better, we should develop a network that takes the previous `T` images as input and outputs the next image as output. Then we can compare the error when performing long time rollouts (e.g. 30 steps as in the paper) to check that the Hamiltonian approach is actually more accurate and faster.

**Comment by OleguerCanal** [comment]
Good idea! Should the network be some kind of RNN to model the evolution?

Maybe also one point of their approach is that you can go back in time, which I'm not sure we could do with a simpler approach
